src/ucp/wireup/wireup_ep.c

```diff
@@ -43,17 +43,125 @@ ucp_wireup_ep_connect_to_ep(uct_ep_h uct_ep, const uct_device_addr_t *dev_addr,
     return uct_ep_connect_to_ep(wireup_ep->super.uct_ep, dev_addr, ep_addr);
 }
 
+static void
+ucp_wireup_tmp_ep_destroy_complete_cb(void *request, ucs_status_t status,
+                                      void *user_data)
+{
+    ucp_wireup_ep_t *wireup_ep = (ucp_wireup_ep_t*)user_data;
+
+    if (wireup_ep == NULL) {
+        /* check for NULL pointer to workaround Coverity warning (it wrongly
```

Comment:
would assert help here instead?

Reply:
yes, will try `ucs_assert_always()`

Reply:
ucs_assert() should be enough to suppress coverity

Reply:
> ucs_assert() should be enough to suppress coverity

yes, but I guess it's OK to have `always`, since it's not a critical path